drools-wb-services/drools-wb-verifier/drools-wb-verifier-web-worker/src/main/resources/org/drools/workbench/verifier/webworker/linker/ScriptTemplate.js

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2016 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+$stats = function() {
```

Comment:
I can throw some comments here. Not something we run into every day. Basically this is just the template with some wiring. 

Or I'll use the linker in Errai, it is pretty much the same thing as this and my WebWorker class, with slightly different wire up. I know there was a worry the web worker codes in Errai might be out dated, but what we do here is just few lines of JavaScript in the main app and linker in the web worker. That can't really get that old. I already use the Errai version for setting up the web worker. The only reason I did not change to Errai here is I was afraid something might break and I wanted this to work fast, but now that I have a 100% working point to go back to the Errai linker might be good, for one it has better support for unexpected exceptions.